docs/content/configuration.md

```diff
@@ -202,6 +207,40 @@ RUN apt-get update && \
 Dump and restore is not supported yet for SQL Server.
 :::
 
+### Default binary options
+
+Apart from the essential options (credentials, database name, etc.), the bundle
+also passes a few default options to the binary depending on the operation being
+performed and the invoked binary itself. You can customize those default options
+by configuring your own ones per operation type and DBAL connection:
```

Comment:
The YAML exemple seem to indicate the contrary: you override options per binary, and not per connection ?

Reply:
Misread, please forget.